pkg/network/prometheus.go

```diff
@@ -34,6 +35,22 @@ var (
 		[]string{"description", "value"},
 	)
 
+	neogoVersion = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Help:      "NeoGo version",
+			Name:      "neogo_version",
```

Comment:
`neogo_neogo_version`?

Reply:
Well, it's just because we have `neo_go` namespace. But you're right, it's too much, fixed.